packages/editor-ui/src/components/Node.vue

```diff
@@ -161,6 +171,49 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.setSubtitle();
 			}
 		},
+		nodePosition: {
+			handler() {
+				// Re-render the helping tooltip for trigger node if its position is changed
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = true;
+					}, 100);
+				}
+			},
+			immediate: true,
+			deep: true,
+		},
+		nodeOffsetPosition: {
+			handler() {
+				// Re-render the helping tooltip for trigger node if the workflow is zoomed-out/in
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = true;
+					}, 100);
+				}
+			},
+			immediate: false,
+			deep: true,
+		},
+		workflowRunning: {
+			handler(bool) {
+				// Show helping tooltip for trigger node if:
+				// #workflow is runnung (bool)
+				// #if it is a trigger node (this.isWebhookNode)
+				// #if it is only one trigger node in the WF (this.isSingleWebhookNode)
+				// #if the node is not disabled (!this.data.disabled)
+				if (bool && this.isWebhookNode() && this.isSingleWebhookNode() && !this.data.disabled) {
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = bool;
```

Comment:
found a bug where trigger does not work... to reproduce:
1. add Webhook node
2. add second webhook node (Webhook1)
3. run workflow and then stop it
4. delete first Webhook node
5. run workflow (tooltip does not show)

Reply:
After re-work, I cannot reproduce this bug anymore with the latest push.